client/packages/system/src/Encounter/DetailView/DetailView.tsx

```diff
@@ -75,9 +76,20 @@ export const DetailView: FC = () => {
   useEffect(() => {
     if (encounter)
       setSuffix(
-        `${
-          encounter.document.documentRegistry?.name
-        } - ${dateFormat.localisedDateTime(encounter.startDatetime)}`
+        <>
+          <Breadcrumb
+            to={RouteBuilder.create(AppRoute.Dispensary)
+              .addPart(AppRoute.Patients)
+              .addPart(encounter.patient.id)
+              .build()}
+            key={'patient'}
+          >
+            {`${encounter.patient.firstName} ${encounter.patient.lastName}`}
+          </Breadcrumb>
+          <span key={'encounter'}>{` / ${
+            encounter.document.documentRegistry?.name
+          } - ${dateFormat.localisedDate(encounter.startDatetime)}`}</span>
+        </>
```

Comment:
to remove the console error:

```suggestion
        <span>
          <Breadcrumb
            to={RouteBuilder.create(AppRoute.Dispensary)
              .addPart(AppRoute.Patients)
              .addPart(encounter.patient.id)
              .build()}
            key={'patient'}
          >
            {`${encounter.patient.firstName} ${encounter.patient.lastName}`}
          </Breadcrumb>
          <span>{` / ${
            encounter.document.documentRegistry?.name
          } - ${dateFormat.localisedDate(encounter.startDatetime)}`}</span>
        </span>
```